lms/djangoapps/instructor/services.py

```diff
@@ -69,3 +71,12 @@ def delete_student_attempt(self, student_identifier, course_id, content_id):
                     )
                 )
                 log.error(err_msg)
+
+    def is_course_staff(self, user, course_id):
```

Comment:
This is an in-proc service endpoint which is called out from edx-proctoring. I can adjust the implementation of this method, to simply delete to user_has_role, to minimize code duplication.